Source/WebKit/Platform/IPC/ArgumentCoders.h

```diff
@@ -94,7 +93,7 @@ template<typename T, size_t Extent> struct ArgumentCoder<std::span<T, Extent>> {
         if constexpr (Extent == std::dynamic_extent)
             return data;
         else
-            return std::span<T, Extent> { data.data(), Extent };
+            return unsafeMakeSpan<T, Extent>(data.data(), Extent);
```

Comment:
I think you can do data.subspan<0, Extent>() here. Slightly nicer not to use the unsafe function.

Reply:
Or maybe data.first<Extent>() is better? I'm new to span API, lol.

Reply:
Going with `data.subspan<0, Extent>()` which seems to work.